contracts/deploy/base/025_base_curve_amo.js

```diff
@@ -48,8 +49,8 @@ module.exports = deployOnBaseWithGuardian(
 
     // Initialize Base Curve AMO implementation
     const initData = cOETHBaseCurveAMO.interface.encodeFunctionData(
-      "initialize(address[],address[])",
-      [[addresses.base.CRV], [addresses.base.WETH]]
+      "initialize(address[],uint256)",
+      [[addresses.base.CRV], oethUnits("0.2")]
```

Comment:
0.2 equals 20%. we should go for 0.2% which would be: 
```
oethUnits("0.002") // 0.2%
```

Reply:
I think it would be cool that we also add a fork test that verifies the value of max slippage.

Reply:
Good catch!

Fixed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2384/commits/615c275192aa52361ed6b9b793708a8825353622